transferui/src/main/res/layout/dialog_fragment_transfer_destination.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/currency_selection_layout_container"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
```

Comment:
this fragment needs progress bar or else this will freeze while getting, transfer method list

Reply:
added here https://github.com/hyperwallet/hyperwallet-android-ui-sdk/pull/84